src/import/config.rs

```diff
@@ -18,13 +18,15 @@ use std::result::Result;
 #[serde(default)]
 #[serde(rename_all = "kebab-case")]
 pub struct Config {
+    pub import_dir: String,
     pub num_threads: usize,
     pub stream_channel_window: usize,
 }
 
 impl Default for Config {
     fn default() -> Config {
         Config {
+            import_dir: "/tmp/tikv/import".to_owned(),
```

Comment:
Why not `data-dir/import`?

Reply:
There is no `data-dir` here, and it is conceptually not related to the tikv `data-dir`.